apps/web/components/manual-feedback-form.tsx

```diff
@@ -0,0 +1,52 @@
+"use client";
+
+import { z } from "zod";
+import { zodResolver } from "@hookform/resolvers/zod";
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from "@repo/ui/components/form";
+import { Textarea } from "@repo/ui/components/textarea";
+import { useForm } from "react-hook-form";
+
+const FormSchema = z.object({
+  feedbacks: z.string().min(30, {
+    message: "Bio must be at least 30 characters.",
+  }),
+});
+
+const ManualFeedbackForm = () => {
+  const form = useForm<z.infer<typeof FormSchema>>({
+    resolver: zodResolver(FormSchema),
+  });
+
+  return (
+    <Form {...form}>
+      <form className="space-y-6">
+        <FormField
+          control={form.control}
+          name="feedbacks"
```

Comment:
> https://english.stackexchange.com/questions/176805/plural-of-feedback

This stupid english 🤷‍♂️

```suggestion
          name="feedback"
```